packages/api/src/schema/schema.yaml

```diff
@@ -811,6 +813,354 @@ components:
+          description: List of NFTs minted with the asset as a source
+          items:
+            type: object
+            properties:
+              chain:
+                type: string
+                description: Chain of the NFT
+                example: eth
+              contract:
+                type: string
+                description: Contract of the NFT
+                example: 0x57f1887a8bf19b14fc0d912b9b2acc9af147ea85
+              token:
+                type: number
+                description: Token id of the NFT
+                example: 123
+
+    task:
+      type: object
+      table: task
+      properties:
+        id:
+          type: string
+          description: Task ID
+          example: 1-23-123-123-1123-123
+        kind:
+          type: string
+          description: Type of the task
+          example: import
+        status:
+          type: string
+          description: Status of the task
+          example: ready
+        createdAt:
+          type: string
+          description: Timestamp (in milliseconds) at which task was created
+          example: 2022-01-01T02:20:00Z
+        updatedAt:
+          type: string
+          description: Timestamp (in milliseconds) at which task was updated
+          example: 2022-01-01T02:20:00Z
+        params:
+          type: object
+          description: Parameters of the task
+          properties:
+            import:
+              type: object
+              description: Parameters for the import task
+              properties:
+                url:
+                  type: string
+                  description: URL of the asset to import
+                  example: https://cdn.livepeer.com/ABC123/filename.mp4
+            export:
+              type: object
+              description: Parameters for the export task
+              properties:
+                kind:
+                  type: string
+                  description: Type of the export
+                  example: arweave|ipfs|s3
+                s3:
+                  type: object
+                  description: Parameters for the S3 export
+                  properties:
+                    destination:
+                      type: string
+                      description: Destination of the export
+                      example: s3://bucket/path/to/file.mp4
+                    key:
+                      type: string
+                      description: Key of the export
+                      example: ABC123
+                    secret:
+                      type: string
+                      description: Secret of the export
+                      example: ABC123
+            transcode:
+              type: object
+              description: Parameters for the transcode task
+              properties:
+                assetId:
+                  type: string
+                  description: ID of the asset to transcode
+                  example: 123123123123
+                preset:
```

Comment:
Should we include transcoding preset adding a Preset resource type in a second step? 

- If yes, should the video and audio transcoding parameters be only in the transcoding preset object and a transcode task takes all the informations from that?
- If no, maybe we should decide which transcoding params are available in a first release

Reply:
I think the properties here should be very similar, or maybe 100% equivalent to the ones from `cli-transcoder`. Let's just copy the CLI flags from it and increment this as it grows.

We shouldn't worry about this task type anyway for the MVP. Let's focus on creating the task infra only with the `import` task for now, and we will create the `export` and `transcode` task types right after.